connector/src/main/java/org/geysermc/connector/network/session/cache/WorldCache.java

```diff
@@ -71,7 +71,8 @@ public int increaseAndGetScoreboardPacketsPerSecond() {
      * @param value True to show, false to hide
      */
     public void setShowCoordinates(boolean value) {
-        showCoordinates = value;
-        session.sendGameRule("showcoordinates", value);
+        boolean check = !session.isReducedDebugInfo() && session.getConnector().getConfig().isShowCoordinates();
```

Comment:
Check? Check what??

Legitimately it's almost 11 PM here and I had to think about this for a minute. I would recommend giving it a more apt name and/or adding a comment above.